### PR TITLE
feat(BUX-000): unifies github actions workflow

### PR DESCRIPTION
**Pull request by wregulski**

### CODE REVIEW

**Review by arkadiuszos4chain — APPROVED** [review]

**Review by pawellewandowski98 — APPROVED** [review]

File: .github/workflows/release.yml
Comment (dorzepowski) [inline .github/workflows/release.yml:1]: I think that you copy older version of the release workflow from go-broadcast-client. Current contain only curl call. (no checkout, no setup go, etc.)

File: Taskfile.yml
Comment (dorzepowski) [inline Taskfile.yml:1]: it also look like older version of taskfile

Comment (wregulski) [reply Taskfile.yml:1]: fixed

File: .github/workflows/release.yml
Comment (wregulski) [reply .github/workflows/release.yml:1]: fixed

**Review by dorzepowski — APPROVED** [review]

**Review by arkadiuszos4chain — APPROVED** [review]

**Pull request merged**